backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -326,3 +326,8 @@ var launcherResources = k8score.ResourceRequirements{
 const (
 	tmplEntrypoint = "entrypoint"
 )
+
+var dummyImages = map[string]bool{
+	"argostub/createpvc": true,
+	"argostub/deletepvc": true,
```

Comment:
I wonder if we can use more arbitrary container names. While, it's a very wild scenario, it's still possible for someone to create a docker hub repo called `argostub` and upload containers `createpvc` and `deletepvc`. WDYT?

Besides, should we add comments and document it somewhere?

Reply:
I agree that these cases may happen, although not with a very high likelihood. These dummy image names seem to be the sole identification that this component is not a normal user defined component, but a special component for a specific function. I think the user will have to avoid such container names and we definitely should make this clear in the docs. I will add comments here and a TODO that we need to include these restrictions in the documentation.